components/grid/editing/built-in-dialogs/overview.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Overview
+page_title: Editing - Built-in Dialogs Overview
+description: Enable built-in dialogs in Grid for Blazor.
+slug: components/grid/features/built-in-dialogs
+tags: telerik,blazor,grid,built,in,dialogs
+published: True
+previous_url: /grid/built-in-dialogs
+position: 0
+---
+
+# Grid Built-in Dialogs
+You can enable built-in dialogs in Grid to provide an extra step for the end-user before a Grid operation.
+
+The Grid component can use built-in dialogs for: 
+
+* [delete confirmation]({%slug grid-delete-confirmation%}) - displays a confirmation dialog when the user clicks the "**delete**" command button.
```

Comment:
There is also the popup editing built-in dialog. Is there a serious reason for this article and its folder?

Reply:
We decided to create a folder for such confirmation dialogs because there is a high chance of adding other confirmation dialogs in the future.